### ISSUE TITLE
Few years of typos

### ISSUE THREAD

**Issue by szepeviktor** [timber/docs#90]
@Levdbas No problem!

Warning: "ligth" should be "light".
Warning: "accessibilty" should be "accessibility".
Warning: "inital" should be "initial".
Warning: "orignally" should be "originally".
Warning: "wheter" should be "whether".
Warning: "aboslute" should be "absolute".
Warning: "retrived" should be "retrieved".

I fix them right away ... (in a PR)
